soc/arm/st_stm32/stm32l4/Kconfig.defconfig.series

```diff
@@ -16,10 +16,5 @@ config SOC_SERIES
 config STM32_LPTIM_TIMER
 	bool
 	default y
-	depends on PM
```

Comment:
why removing "depends on PM" there

Reply:
Not related indeed. Don't know how this ended up here. Will fix